src/db/db.sql

```diff
@@ -159,8 +159,15 @@ CREATE TABLE IF NOT EXISTS access (
 	flags TEXT
 );
 
+-- Holds bans that the bot is tracking (due to sanctions)
+CREATE TABLE bantrack (
+	player INTEGER NOT NULL PRIMARY KEY REFERENCES player(id) DEFERRABLE INITIALLY DEFERRED,
+	expires DATETIME,
+	warning_amount INTEGER
+);
+
 -- Used to hold state between restarts
-CREATE TABLE IF NOT EXISTS pre_restart_state (
+CREATE TABLE pre_restart_state (
 	-- List of players to ping after the bot comes back online
```

Comment:
Well in any case there's no guarantee that the users wouldn't run it themselves. But since this wouldn't delete data, then it's fine.